README.md

```diff
@@ -128,17 +128,18 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
 
 The tool is provided as a unique `swiftgen` binary command-line, with the following subcommands:
 
-* `swiftgen assets [OPTIONS] DIR`
-* `swiftgen colors [OPTIONS] FILE`
-* `swiftgen fonts [OPTIONS] DIR`
-* `swiftgen storyboards [OPTIONS] DIR`
-* `swiftgen strings [OPTIONS] FILE`
+* `swiftgen colors [OPTIONS] FILE1 …`
+* `swiftgen fonts [OPTIONS] DIR1 …`
+* `swiftgen storyboards [OPTIONS] DIR1 …`
+* `swiftgen strings [OPTIONS] FILE1 …`
+* `swiftgen xcassets [OPTIONS] FILE1 …`
```

Comment:
Yeah could be a good idea.
But after thinking about it a bit, it's probably not worth it. We want the `Usage` summary (the 1-line invocation template) to be short and comprehensible.

If people need more info (about if they have to use a DIR containing asset catalogs or the PATH to the assets catalog itself or whatnot), they can use `swiftgen xcassets --help`. So maybe that's not worth bikeshedding over this…